cmd/envtool/setup.go

```diff
@@ -172,6 +172,12 @@ func setup(ctx context.Context, logger *slog.Logger) error {
 		return lazyerrors.Error(err)
 	}
 
+	// TODO https://github.com/FerretDB/FerretDB/issues/5369
+	if !yugabyteDB {
+		logger.InfoContext(ctx, "Skipping YugabyteDB setup")
+		return nil
+	}
```

Comment:
That means we will not print the final 'Done' below. I think that's the case where early return is not warranted.